content/Reducing_Power_Consumption.md

```diff
@@ -59,14 +59,25 @@ power standby mode. For example, a program may use this feature when it
 hasn't sensed keyboard input for a longer period (for example, when
 somebody forgot to turn off the Game Boy).
 
-Before invoking STOP, it might be required to disable Sound and Video
-manually (as well as IR-link port in CGB). Much like HALT, the STOP
-state is terminated by interrupt events. STOP is commonly terminated
-with a joypad interrupt.  A bug may keep STOP from terminating if STOP
-was invoked while a button was held.
-
-During STOP mode, the display will turn white, so avoid using it in your
-game's main loop.
+No licenced rom makes use of STOP outside of CGB speed switching.
```

Comment:
We are currently using the `license` spelling, can you keep that?

In case you want to switch to the British variant we you can open an Issue about that.

Reply:
I didn't even see that there are two ways to spell license, so no hidden meaning.